huaweicloud/services/ddm/resource_huaweicloud_ddm_instance.go

```diff
@@ -172,6 +173,23 @@ func ResourceDdmInstance() *schema.Resource {
 					"true", "false",
 				}, false),
 			},
+			"read_weight": {
```

Comment:
can we use this parameter in while creating the instance?

Reply:
no, only can be put in update function. read_weight depends on ddm_instance and ddm_schema, ddm_schame depends on ddm_instance.